### PR TITLE
added judicial api yaml

### PR DESCRIPTION
**Pull request by reem-hmcts**
### Change description ###

added judicial api yaml

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: k8s/aat/common/rd/judicial-api.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: rd-judicial-api
+    version: 0.0.2
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/rd/judicial-api:prod-e576d62a
```
Comment (shashank-rastogi) [inline k8s/aat/common/rd/judicial-api.yaml]: this should be version of judicial  api image in hmctspublic

File: CODEOWNERS [hunk 2]
```diff
@@ -119,6 +119,7 @@ k8s/**/common/rd/identity.yaml @hmcts/ref-data
 k8s/**/common/rd/professional-api.yaml @hmcts/ref-data
 k8s/**/common/rd/profile-sync.yaml @hmcts/ref-data
 k8s/**/common/rd/user-profile-api.yaml @hmcts/ref-data
+k8s/**/common/rd/judicial-api.yaml @hmcts/ref-data
```
Comment (timja) [inline CODEOWNERS]: this should be AAT until its approved for prod
```suggestion
k8s/aat/common/rd/judicial-api.yaml @hmcts/ref-data
```

Comment (reem-hmcts) [reply CODEOWNERS]: Thanks, have amended

File: k8s/aat/common/rd/judicial-api.yaml (hunk 1 above)
Comment (timja) [inline k8s/aat/common/rd/judicial-api.yaml]: This is an image that exists? If it doesn't just put latest and flux will pickup the latest one, if it doesn't exist flux will fail to apply the change

File: k8s/aat/common/rd/judicial-api.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: rd-judicial-api
+    version: 0.0.2
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/rd/judicial-api:prod-latest
```
Comment (timja) [inline k8s/aat/common/rd/judicial-api.yaml]:
```suggestion
      image: hmctspublic.azurecr.io/rd/judicial-api:latest
```

Comment (reem-hmcts) [reply k8s/aat/common/rd/judicial-api.yaml]: ok, apologies, should be right now

**Pull request merged**